buncha docs (#13)
theopenlane/openlane-docs: docs/docs/docs/platform/standards/soc2/security-management-procedures.mdx

As of this writing, businesses are experiencing something known as the great resignation. This has never occurred before throughout history and going forward I think employers should do more to excite and retain employees, because once an employee leaves the organization. The knowledge regarding the projects they worked on and contributions made to the organization also leaves with them.

#### Onboarding & Training

Once the human resources team has selected its candidate and everyone on the team feels comfortable. It's the responsibility of the human resources department to ensure that an onboarding process is in place and followed. The onboarding process should include timely communication with the employees and also an introduction to the organization either through email or instant messaging platforms such as Slack, or Microsoft teams.

The onboarding process should also include the necessary training for a newly hired employee. Including what the company does as a whole, what processes they will own, and an overview of the reporting structure of the company.

> This is also a great time for the organization to have the employee review any company policies and gather a signature. A lot of times these processes can be automated with the organization's HR solution. If it's not available then a manual process will have to be completed. This is also a great time for the organization to ensure employees are made aware of basic job responsibilities and processes within the team or department.

Reviewer: repetitive "This is also a great time for" twice in this short paragraph

Tip - SOC 2 Auditors will pull samples for requested training, it's vital to have this information available for the auditors during the audit period.

**Example Training List**

1. Company Overview
1. Platform training
1. Security awareness training
1. Software development lifecycle training (SDLC) if applicable
1. Safety Training
1. Financial securities training if applicable

#### Transitioning positions within an organization

> As the business grows, the human resources team will find itself trying to scale a business, and with any new product or service on the market. It will have a tough time finding employees that have the right skills and experience to fill more senior-level positions. With that being said, internal promotions are going to happen.

Reviewer: ```suggestion
As the business grows, the human resources team will find itself trying to scale a business, and with any new product or service on the market, it will have a tough time finding employees that have the right skills and experience to fill more senior-level positions. With that being said, internal promotions are going to happen.
```

Sometimes, employees will change departments completely. Just like when a new employee was hired and the human resources team announced it. The human resources team needs to announce when an employee changes positions. This will allow the employees to be notified but also from a security perspective, reassign processes and roles based upon the job duties of the position.

> At a high level, there should be a ticketing system in place that human resource managers submit employee position changes to. On the backend, “ticketing side” there should be a process to re-evaluate the employees' access roles within the environment. Typically the change is completed by the Information Technology department with the addition of more access approved by the manager over the employee. Please see the basic process flow attached below.

Reviewer: "Please see the basic process flow attached below." 

Where is this?

#### Offboarding and Terminations

> All companies will face offboarding and terminations at some point. During this phase even if you're a small startup. Offboarding procedures need to be in place. They need to be defined and followed. This is a critical step, as a lot of organizations will not remove access days or even weeks after an employee has been terminated. This is simply because there is no defined process and the lack of communication between Human Resources and Information technology departments did not occur.

Reviewer: ```suggestion
All companies will face off-boarding and terminations at some point. During this phase, even if you're a small startup, off-boarding procedures need to be in place. They need to be defined and followed. This is a critical step, as a lot of organizations will not remove access days or even weeks after an employee has been terminated. This is simply because there is no defined process and the lack of communication between Human Resources and Information technology departments did not occur.
```

**Example offboarding procedures**

1. Human Resources is notified that an employee is leaving or should be terminated.
1. Human Resources submits a ticket notifying Information Technology to remove access on desired date and time.
1. Information Technology receives requests and plans accordingly.
1. The former employee is termed and access is cut.
1. Communication to the organization notifying departments or teams that the employee has been offboarded.

#### Annual Evaluation

One of the most critical things for an employee to receive at the beginning, or at the end of each year is their annual evaluation. This is important not only for the employee but also for the organization to continue developing its employees and giving them a chance to bring attention to issues that could be plaguing the organization but not known at the management level.

Not only is the annual evaluation good for both the employee and employer, but it's also a metric used for control within the SOC 2 framework. The annual evaluation can follow any format, as long as they are recorded and the feedback is provided back to the employee.

> For organizations that do not believe in an annual evaluation because they have weekly 1:1 sessions. This is also acceptable, however, you will want to ensure that the managers are maintaining notes and at a high level at least performing yearly reviews that touch base on projects that were completed by the team and individual contributions made by the employee.

Reviewer: ```suggestion
For organizations that do not believe in an annual evaluation because they have weekly 1:1 sessions, this is also acceptable. However, you will want to ensure that the managers are maintaining notes and at a high level at least performing yearly reviews that touch base on projects that were completed by the team and individual contributions made by the employee.
```

**Audit Steps**

There are several ways you could audit this. It will vary from organization to organization. However, the key points are outlined below.

1. Is logging enabled for all systems, applications, services, databases that are in scope of the platform or system
1. Review the architecture diagram that has been provided
1. Request a sample of logs from the diagram that outlines applications, services, databases
1. If a SIEM is used, request to see a population of systems that are pushing logs into the SIEM

### Data Breaches and Disclosures

From the previous modules we have learned that you as a lead implementer will be asking the organization a lot of questions around their processes, people and technology. Now we will be discussing ethics around data breaches, data disclosures, and your role as a lead implementer or consultant.

> Data has been extracted

Reviewer: Is this supposed to be a header?

**Examples of Coverage that is included for Cyber Liability Insurance**

1. Public relations communications
1. Incident response resources
1. Assistance for recovery and business continuity

**Audit Steps**

1. Perform a tabletop exercise
1. Notate deficiencies during the exercise
1. Assist the organization in the creation of a data breach playbook
1. Ensure defined roles are known to all parties
1. Ensure the playbook is updated yearly

**Data Disclosure**

We all know organizations share data, they buy and sell our data. We have seen this occur with social tech giants like Facebook, now known as Meta. Google with their advertising and tracking ids. However many companies also buy and sell our data or share it with their partners.

They do this for multiple reasons. The most common being it provides organizations monetary value or for data enrichment purposes.

With that being said, it's important to know the following.

1. Does the organization sell its customers' data?

> 1.

Reviewer: ```suggestion
```

   * If yes, to whom?
1. Are customers made aware that this behavior occurs?
1. Does the data disclosure list the companies it provides data with?

> 1.

Reviewer: ```suggestion
```

   * If not how does the company define the term "Parties or Third Parties"

Audit Steps

1. Audit the privacy policy

> 1.

Reviewer: ```suggestion
```

Reviewer: weird formatting/bullets here: 
<img width="594" alt="image" src="https://github.com/user-attachments/assets/ea9759f7-6a31-48aa-8659-f8338dbef8c9" />

   * Does it outline the data the platform captures and processes?
1. Are Data Processing Agreements "DPA's" in place with contracts?
1.
   * You will need to pull a list of contracts and DPA's
   *
     * Important Note - DPA's are not always needed
1. Review a sample of that data that was sold, or provided to a third party.
1.
   * Do the data fields align with what is defined in the data disclosure?

> 1. Ask for a population of third parties the data was shared with'''

Reviewer: ```suggestion
1. Ask for a population of third parties the data was shared with
```

  * Develops the change
  * Test the change
  * Implements the change
* Testing strategies should be formalized and implemented. All changes should ideally be tested prior to implementation. This includes functional testing, UAT testing, QA testing etc.
* All changes must be approved and authorized prior to implementation. This should be a formal sign off or authorization on a ticketing tool. This is normally a key piece of evidence for the auditors.
* An emergency change process should also be implemented and clearly defined in the change management policy.
* Separate environments of a system should exist:
  * Development environment
  * QA/ test environment
  * Production environment

The above is not a conclusive list of how a change management process should look like, but these are the key elements.

**Audit Steps**

1. Select one change and perform a walkthrough with management, so that the auditors can get an understanding of the change management process.
1. Pull a listing of changes for the period of review from the ticketing tool or the system itself
1. Select a sample of changes and request the following evidence:

> 1.

Reviewer: ```suggestion
```
